### PR TITLE
Copy edit Section 2 intro, fixes #302.

### LINKED ISSUE #302: Copy edit beginning of section 2 through 2.1
2. Principles for Privacy on the Web
2.1 Identity on the Web
2.1.1 Recognition Types
2.1.2 User agent awareness of recognition
2.1.3 Recognition Methods

### PR DESCRIPTION
**Pull request by rhiaro**
Moves extra text about context to a Context section in the Common Concepts appendix.

See https://github.com/w3ctag/privacy-principles/pull/321 for edits in context.

***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/pull/322.html" title="Last updated on Jul 5, 2023, 4:15 PM UTC (a26e28a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/322/077d2a9...a26e28a.html" title="Last updated on Jul 5, 2023, 4:15 PM UTC (a26e28a)">Diff</a>

### CODE REVIEW

**Review by darobin — APPROVED** [review]
Big improvement, thanks.

File: index.html [hunk 1]
```diff
@@ -2126,9 +2122,15 @@
 are people in some situations of intellectual or psychological impairment, are
 refugees, etc.
 
+## Context {#context}
```
Comment (jyasskin) [inline index.html:2125]:
```suggestion
## Contexts {#context}
```

To match the other section titles here.

File: index.html [hunk 2]
```diff
@@ -2126,9 +2122,15 @@
...
+
 A <dfn>context</dfn> is a physical or digital environment in which [=people=] interact with other
 [=actors=], and which the [=people=] understand as distinct from other [=contexts=].
 
+A [=context=] is not defined in terms of who owns or controls it. Sharing
+[=data=] between different [=contexts=] of a single company is just as much
+a [=privacy violation=] as if the same data were shared between unrelated [=actors=].
```
Comment (rhiaro) [inline index.html:2132]:
```suggestion
A [=context=] is not defined in terms of who owns or controls it. Sharing
[=data=] between different [=contexts=] of a single company is
a [=privacy violation=], as is the same data were shared between unrelated [=actors=].
```

File: index.html [hunk 3]
```diff
@@ -1023,17 +1023,13 @@
...
-fully expect more specific [=contexts=] of the Web to add their own principles to further constrain
-[=information flows=].
-
-To the extent possible, [=user agents=] are expected to enforce these principles. However, this is not
-always possible and additional enforcement mechanisms are needed. One particularly salient issue
-is that a [=context=] is not defined in terms of who owns or controls it. Sharing
-[=data=] between different [=contexts=] of a single company is just as much a [=privacy violation=] as
-if the same data were shared between unrelated [=actors=].
+This section describes a set of principles designed to apply to the web
+[=context=] in general. More specific [=contexts=] on the web may need more
```
Comment (jyasskin) [inline index.html:1027]:
```suggestion
[=context=] in general. Specific [=contexts=] on the web may need more
```

**Review by jyasskin — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (jyasskin) [inline index.html:2132]:
```suggestion
[=data=] between different [=contexts=] of a single company is
a [=privacy violation=], just as if the same data were shared between unrelated [=actors=].
```

**Pull request merged**